src/main/scala/sbtdatabricks/DatabricksHttp.scala

```diff
@@ -142,6 +142,148 @@ class DatabricksHttp(endpoint: String, client: HttpClient, outputStream: PrintSt
+    val post = new HttpPost(endpoint + COMMAND_EXECUTE)
+    val entity = new MultipartEntity()
+
+    entity.addPart("language", new StringBody(language.is))
+    entity.addPart("clusterId", new StringBody(cluster.id))
+    entity.addPart("contextId", new StringBody(contextId.id))
+    entity.addPart("command", new FileBody(commandFile))
+    post.setEntity(entity)
+
+    val response = client.execute(post)
+    val handler = new BasicResponseHandler()
+    val responseString = handler.handleResponse(response).trim
+    outputStream.println(s"Execute Command: Received the following response: '${responseString}'")
+    mapper.readValue[CommandId](responseString)
+  }
+
+  /**
+   * Check the status of a command
+   * @param cluster the relevant cluster within which the context will be created
+   * @param contextId The id of the execution context
+   * @param command The code to be executed on the cluster
+   * @return The status of the command
+   *
+   */
+  private[sbtdatabricks] def checkCommand(
+      cluster: Cluster,
+      contextId: ContextId,
+      commandId: CommandId): CommandStatus = {
+    outputStream.println(s"Checking status of command '${commandId.id}' on cluster '${cluster.name}'")
+    val form =
+      URLEncodedUtils.format(List(new BasicNameValuePair("clusterId", cluster.id),
+                                  new BasicNameValuePair("contextId", contextId.id),
+                                  new BasicNameValuePair("commandId", commandId.id)), "utf-8")
+    val request = new HttpGet(endpoint + COMMAND_STATUS + "?" + form)
+    val response = client.execute(request)
+    val handler = new BasicResponseHandler()
+    val responseString = handler.handleResponse(response).trim
+    outputStream.println(s"Check Command: Received the following response: '${responseString}'")
+    mapper.readValue[CommandStatus](responseString)
+  }
+
+  /**
+   * Cancel a command
+   * @param cluster the relevant cluster within which the context will be created
+   * @param contextId The id of the execution context
+   * @param command The code to be executed on the cluster
+   * @return The id of the command
+   *
+   */
+  private[sbtdatabricks] def cancelCommand(
+      cluster: Cluster,
+      contextId: ContextId,
+      commandId: CommandId): CommandId = {
+    outputStream.println(s"Cancelling command '${commandId.id}' on cluster '${cluster.name}'")
+    val post = new HttpPost(endpoint + COMMAND_CANCEL)
+    val request = CancelCommandRequestV1(cluster.id, contextId.id, commandId.id)
+    val form = new StringEntity(mapper.writeValueAsString(request))
+    form.setContentType("application/json")
+    post.setEntity(form)
+    val response = client.execute(post)
+    val handler = new BasicResponseHandler()
+    val responseString = handler.handleResponse(response).trim
+    outputStream.println(s"Cancel Command: Received the following response: '${responseString}'")
```

Comment:
I'm wondering whether these are required. Showing the user the `CommandId` is not very descriptive or useful.